doc/examples/etc_cloud_cfg.txt

```diff
@@ -0,0 +1,184 @@
+# The top level settings are used as module
```

Comment:
Great consolidation of doc content here. It's appropriate and gives context to the intent of system/base configuration.

Since we are filtering this out of cloud-config examples page, is there a good home we can find this example in `doc/rtd/reference/base_config_reference.rst`. Maybe in belongs down under the "Example" section of that page? There is a lot of duplication between this content and the example in the current doc.